bgpd/bgp_route.c

```diff
@@ -11363,6 +11363,9 @@ static int bgp_table_stats(struct vty *vty, struct bgp *bgp, afi_t afi,
 	if (json_array)
 		json = json_object_new_object();
 
+	if (!bgp)
```

Comment:
I don't think this is needed with the second commit right?

Reply:
right

Reply:
removed